### PR TITLE
Added support for renewal-api, order-api and booking-api

### PR DESCRIPTION
**Pull request by Clairevf**

### CODE REVIEW

File: pkg/bookings/bookings.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+	"github.com/clearchanneloutdoor/io-sdk-golang/pkg/clients"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+const (
+	serverUrl string = "https://direct.cco.io"
+)
+
+func NewClient(ctx context.Context, oauth2 *clientcredentials.Config, overrideSvr ...string) (*clients.Client[Booking], error) {
```
Comment (cco-robin) [inline pkg/bookings/bookings.go:14]: don't forget to reference these new clients in cmd/main.go

File: pkg/orders/orders.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	return clients.NewClient[Order](ctx, svr, "/v1/orders", oauth2)
+}
+
+func NewOrderLinesClient(ctx context.Context, oauth2 *clientcredentials.Config, overrideSvr ...string) (*clients.Client[OrderLine], error) {
+	svr := serverUrl
+	if len(overrideSvr) > 0 && overrideSvr[0] != "" {
+		svr = overrideSvr[0]
+	}
+
+	return clients.NewClient[OrderLine](ctx, svr, "/v1/orderlines", oauth2)
```
Comment (cco-robin) [inline pkg/orders/orders.go:29]: do any of these need any scopes added? not sure what the use case is so they might not

**Review by cco-robin — COMMENTED** [review]
i think once you pull these into cmd/main.go then you should be good!

File: pkg/bookings/bookings.go (hunk 1 above)
Comment (Clairevf) [reply pkg/bookings/bookings.go:14]: oops i'll add that in now!

File: pkg/orders/orders.go (hunk 2 above)
Comment (Clairevf) [reply pkg/orders/orders.go:29]: We're not currently enforcing any scope on order-api or booking-api... some oversight on our end but it's coming down the line. i'll update these when we start to enforce them.

File: pkg/bookings/bookings.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import (
+	"context"
+
+	"github.com/clearchanneloutdoor/io-sdk-golang/pkg/clients"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+const (
+	serverUrl string = "https://direct.cco.io"
```
Comment (cco-joshua) [inline pkg/bookings/bookings.go:11]: This is good, we'll modify the pxy-api clients.yaml for each environment to include the order-api and booking-api... for your immediate needs, however, when using the SDK, you'll want to use the variadic field on NewClient (line 14 below) to provide the Istio address of the API you're hitting:

```golang
bc, err := bookings.NewClient(ctx.Background(), oaCfg, "booking-api.booking-api.svc")
if err != nil {
  // argh...
}
```

**Review by cco-joshua — APPROVED** [review]

**Pull request merged**